app/locations/none.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+
+const {
+  computed: { reads },
+  inject: { service },
+  get
+} = Ember;
+
+const TEMPORARY_REDIRECT_CODE = 307;
```

Comment:
I think we should just use 302 here. 307 was added so that servers can require subsequent requests after redirect responses to use the same request method as the original request that resulted in the redirect. While we want the same request method to be used of course, 307 was really only added for everythong *other* than `GET` and FastBoot is only ever dealing with `GET` anyway (in fact all other request methods expect for `HEAD` maybe) should actually be ignored by the FastBoot middleware - not sure whether that's actually the case yet.